src/styles/utils/normalize.scss

```
@@ -0,0 +1,9 @@
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+}
```

it's optional, but i think you can refactor it to smth like that:

```suggestion
body,
ul {
  margin: 0;
}

ul {
  padding: 0;
  list-style: none;
}
```

`<li>`'s will inherit the `<ul>`'s list-style and you will not reset unnecessary padding for body